src/main/docs/guide/mongo/mongoQuickStart.adoc

```diff
@@ -113,6 +113,15 @@ You can now perform CRUD (Create, Read, Update, Delete) operations on the entity
 
 snippet::example.BookRepositorySpec[project-base="doc-examples/mongo-example", tags="inject", indent="0"]
 
```

Comment:
@dstepanov Does this look ok?

Reply:
Can you please move it above after the first mention of the configuration?

Reply:
Did you push?

Reply:
I did now, sorry. I was on vpn and maybe it didn't get published.